internal/pkg/action/operator_install.go

```diff
@@ -81,6 +81,21 @@ func (i *OperatorInstall) Run(ctx context.Context) (*v1alpha1.ClusterServiceVers
 	return csv, nil
 }
 
+func (i *OperatorInstall) operatorGroupSupportedInstallModes(og v1.OperatorGroup) sets.String {
```

Comment:
Rather than adding an entirely new function, it would be better to update the `possibleInstallModes` function to handle the case where `os.Status.Namespaces[0] == ""`

Something like:
```go
        case 1:
                switch watchNamespaces[0] {
                case "":
                        return sets.NewString(string(v1alpha1.InstallModeTypeAllNamespaces))
                case i.config.Namespace:
                        return sets.NewString(string(v1alpha1.InstallModeTypeOwnNamespace))
                default:
                        return sets.NewString(string(v1alpha1.InstallModeTypeSingleNamespace))
                }
```